### PR TITLE
Move from hard-coded to N-generic attr printing

### PR DESCRIPTION
**Pull request by csuwildcat**
This is a commit that addresses the need for more generic attribute printing to capture all attributes in the remote document's nodes without hard-coding against specific attributes. This feature addresses issue https://github.com/MicrosoftDX/Vorlonjs/issues/50.

### CODE REVIEW

File: Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts [hunk 1]
```diff
@@ -370,19 +373,12 @@ module VORLON {
...
-            if (receivedObject.classes) {
-                this._appendSpan(link, "nodeAttribute", " class");
-                this._appendSpan(link, "nodeTag", "=\"");
-                this._appendSpan(link, "nodeValue", receivedObject.classes);
-                this._appendSpan(link, "nodeTag", "\"");
-            }
+            receivedObject.attributes.forEach(function(attr){
+                var node = document.createElement('span');
+                node.className = 'nodeAttribute';
+                node.innerHTML = '<span>' + attr[0] + '</span><span>' + attr[1] + '</span>';
```
Comment (deltakosh) [inline Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts]: you are not generating ="...", correct?

Comment (csuwildcat) [inline Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts]: Correct, this prints the entire content of all attributes - is that what's
desired?

Comment (deltakosh) [inline Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts]: I would like to have the same display as f12 dom explorer.
Meaning:
<a href="bla" bla="bla" ...>

So with = and "

Comment (csuwildcat) [inline Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts]: Yes, this definitely prints the ="" for all names and values. Can you/do you want to pull in an take a look in a separate branch? Or do you want me to add a screenshot to this issue thread?

Comment (deltakosh) [inline Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts]: OMG, I did not see the CSS rule :)
SO that's perfect! Thank you Daniel!

**Comment by csuwildcat** [comment]
@deltakosh here's a screenshot, I just repeated a style attribute manually to show you how things display and wrap when there is a lot of attribute content: 
attr-ui

**Comment by csuwildcat** [comment]
@deltakosh the reason you're not seeing `=` and `"` chars in the JS, is because they're now automatically added via CSS pseudo elements (which is far better in terms of tree complexity and having to step around them when we add code later).

**Pull request merged**